src/main/java/de/tum/in/www1/artemis/domain/metis/Reaction.java

```diff
@@ -0,0 +1,90 @@
+package de.tum.in.www1.artemis.domain.metis;
+
+import java.time.ZonedDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.DomainObject;
+import de.tum.in.www1.artemis.domain.User;
+
+/**
+ * A Reaction on a Posting.
+ */
+@Entity
+@ReactionConstraints
+@Table(name = "reaction")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class Reaction extends DomainObject {
+
+    @ManyToOne
+    private User user;
+
+    @Column(name = "creation_date")
+    private ZonedDateTime creationDate;
+
+    @Column(name = "emoji_id")
+    private String emojiId;
```

Comment:
is the emoji_id basically the name of the emoji? Or what is the reason you use a string here :D ? I guess emoji_name could be an alternative? I don't think it's that important though

Reply:
it's an identifying, unique name. I see why you found this confusing. I added a doc string to avoid misunderstandings.